src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -306,45 +306,66 @@ This resets the cluster to its initial state.`,
 				return err
 			}
 
-			var eg errgroup.Group
+			forward := func() error {
 
-			eg.Go(func() error {
-				stdin := strings.NewReader(fmt.Sprintf(`
+				var eg errgroup.Group
+
+				eg.Go(func() error {
+					fmt.Println("Forwarding the pachd (Pachyderm daemon) port...")
+					stdin := strings.NewReader(fmt.Sprintf(`
 pod=$(kubectl %v get pod -l app=pachd | awk '{if (NR!=1) { print $1; exit 0 }}')
 kubectl %v port-forward "$pod" %d:650
 `, kubeCtlFlags, kubeCtlFlags, port))
-				return cmdutil.RunIO(cmdutil.IO{
-					Stdin:  stdin,
-					Stderr: os.Stderr,
-				}, "sh")
-			})
-
-			eg.Go(func() error {
-				stdin := strings.NewReader(fmt.Sprintf(`
+					if err := cmdutil.RunIO(cmdutil.IO{
+						Stdin:  stdin,
+						Stderr: os.Stderr,
+					}, "sh"); err != nil {
+						fmt.Println("Could not forward Pachd port")
+						return fmt.Errorf("Could not forward Pachd port")
+					}
+					return nil
+				})
+
+				eg.Go(func() error {
+					stdin := strings.NewReader(fmt.Sprintf(`
 pod=$(kubectl %v get pod -l app=dash | awk '{if (NR!=1) { print $1; exit 0 }}')
 kubectl %v port-forward "$pod" %d:8080
 `, kubeCtlFlags, kubeCtlFlags, uiPort))
-				if err := cmdutil.RunIO(cmdutil.IO{
-					Stdin: stdin,
-				}, "sh"); err != nil {
-					return fmt.Errorf("UI not enabled, deploy with --dashboard")
-				}
-				return nil
-			})
+					if err := cmdutil.RunIO(cmdutil.IO{
+						Stdin: stdin,
+					}, "sh"); err != nil {
+						return fmt.Errorf("Could not forward dash websocket port.")
+					}
+					return nil
+				})
 
-			eg.Go(func() error {
-				stdin := strings.NewReader(fmt.Sprintf(`
+				eg.Go(func() error {
+					fmt.Printf("Forwarding the dash (Pachyderm dashboard) UI port to http://localhost:%v ...\n", uiPort)
+					stdin := strings.NewReader(fmt.Sprintf(`
 pod=$(kubectl %v get pod -l app=dash | awk '{if (NR!=1) { print $1; exit 0 }}')
 kubectl %v port-forward "$pod" %d:8081
 `, kubeCtlFlags, kubeCtlFlags, uiWebsocketPort))
-				cmdutil.RunIO(cmdutil.IO{
-					Stdin: stdin,
-				}, "sh")
+					if err := cmdutil.RunIO(cmdutil.IO{
+						Stdin: stdin,
+					}, "sh"); err != nil {
+						fmt.Printf("Is the dashboard deployed? If not, deploy with \"pachctl deploy local --dashboard-only\"")
+						return fmt.Errorf("Could not forward dash UI port.")
+					}
+					return nil
+				})
+
+				fmt.Println("CTRL-C to exit")
+
+				if err := eg.Wait(); err != nil {
+					return err
+				}
 				return nil
-			})
+			}
```

Comment:
What's the purpose in wrapping this in an inline `func` rather than just calling these things inline? Also if you do still want to do the inline `func` you can call it directly inline, i.e. 

```go
if err := func() error {
    ...
}(); err != nil {
    ...
}